althea_types/src/wifi_info.rs

```diff
@@ -47,6 +47,12 @@ pub struct WifiStationData {
     pub current_time_ms: Option<u64>,
 }
 
+/// Format client uses to send to optools and frontend
+#[derive(Debug, Clone, Serialize, Deserialize, Default)]
+pub struct ClientExtender {
+    pub mac_addr: String,
```

Comment:
don't we already have a mac address type somewhere? If not a number less than 2^48 would be a more restricted representation than a string, which could be literally anything.